dashboard/src/locales/messages/index.ts

```diff
@@ -63,6 +63,9 @@ export const messages = {
       'Success - builds completed successfully{br}' +
       'Failed - builds failed{br}' +
       "Inconclusive - builds with unknown status, including ongoing builds that didn't finish yet.",
+    'codeBlock.highlights': 'Highlights:',
+    'codeBlock.highlightsTooltip':
+      'Highlighted items accounts for any lines citing fails or errors.',
```

Comment:
I think you can give a little more in depth explanation for the logic here

Reply:
Sure, I changed the message adding another line of explanation